geometry_msgs/PoseStamped initial_pose
bool set_initial_pose
---
bool status
